handlers/CNAlifestyleHandler/__init__.py

```diff
@@ -0,0 +1,39 @@
+import json
+import requests
+
+from bs4 import BeautifulSoup
+
+from comment import Comment
+from handlers.AbstractBaseHandler import AbstractBaseHandler, HandlerError
+
+class CNAlifestyleHandler(AbstractBaseHandler):
+    @classmethod
+    def handle(cls, url):
+        html = requests.get(url).text
+        soup = BeautifulSoup(html, "html.parser")
+        article_info = soup.find("meta", {"name":"cXenseParse:mdc-targeturl"})["content"].split('/')
+        category_id = article_info[1]
+        article_id = article_info[2]
+        CNALIFESTYLE_API_URL = f"https://cnalifestyle.channelnewsasia.com/graphql?query=query%20article(%24articleId%3A%20String!%2C%20%24categoryId%3A%20String!)%20%7B%0A%20%20article(articleId%3A%20%24articleId%2C%20categoryId%3A%20%24categoryId)%20%7B%0A%20%20%20%20id%0A%20%20%20%20title%0A%20%20%20%20metaTitle%0A%20%20%20%20image%0A%20%20%20%20imageWidth%0A%20%20%20%20imageHeight%0A%20%20%20%20category%0A%20%20%20%20date%0A%20%20%20%20sharing%0A%20%20%20%20exclusive%0A%20%20%20%20link%0A%20%20%20%20hasSubjectTaxonomy%0A%20%20%20%20currentContext%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20link%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20author%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20contexts%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20link%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20defaultPhoto%20%7B%0A%20%20%20%20%20%20link%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20photos%20%7B%0A%20%20%20%20%20%20teaserText%0A%20%20%20%20%20%20detailText%0A%20%20%20%20%20%20photo%20%7B%0A%20%20%20%20%20%20%20%20link%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20video%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20ooyalaId%0A%20%20%20%20%20%20duration%0A%20%20%20%20%20%20playerId%0A%20%20%20%20%20%20category%0A%20%20%20%20%20%20sk%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20ner%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20audio%20%7B%0A%20%20%20%20%20%20duration%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20html%0A%20%20%20%20teaserText%0A%20%20%20%20contentLength%0A%20%20%20%20postedDate%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D%0A&operationName=article&variables=%7B%22categoryId%22%3A%22{category_id}%22%2C%22articleId%22%3A%22{article_id}%22%7D"
```

Comment:
Can this be separated into multiline f-strings to align with PEP-8? I understand this is the nature of the URL, but Im thinking if multi-line f-strings would look neater in this case

Reply:
I agreed and have separated that into multi-line f-strings.

Reply:
Thank you!

If @fterh has no objections, I will merge this into develop.